src/test/java/de/adorsys/sdjwt/TimeClaimVerifierTest.java

```diff
@@ -0,0 +1,78 @@
+package de.adorsys.sdjwt;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import de.adorsys.sdjwt.exception.SdJwtVerificationException;
+import org.junit.Test;
+
+import static org.junit.Assert.assertThrows;
+
+public class TimeClaimVerifierTest {
+
+    // 60 seconds of leeway
+    private final TimeClaimVerifier timeClaimVerifier = new FixedTimeClaimVerifier(60);
+
+    private static final long CURRENT_TIMESTAMP = 1609459200L; // Fixed timestamp: 2021-01-01 00:00:00 UTC
+
+    static class FixedTimeClaimVerifier extends TimeClaimVerifier {
+
+        public FixedTimeClaimVerifier(int leewaySeconds) {
+            super(leewaySeconds);
+        }
+
+        @Override
+        public long currentTimestamp() {
+            return CURRENT_TIMESTAMP;
+        }
+    }
+
+    @Test
+    public void testVerifyExpClaimExpired() {
+        ObjectNode payload = SdJwtUtils.mapper.createObjectNode();
+        payload.put("exp", CURRENT_TIMESTAMP - 100); // Expired 100 seconds ago
+
+        assertThrows(SdJwtVerificationException.class, () -> timeClaimVerifier.verifyExpClaim(payload));
+    }
+
+    @Test
+    public void testVerifyExpClaimValid() throws SdJwtVerificationException {
```

Comment:
Is this method suppose to throw any exception?, if yes why not throwing an the exception to this method!

Reply:
As a checked exception `SdJwtVerificationException` that can be thrown from the methods called, Java requires this explicit syntax in the case a try/catch block is not used.